packages/notification/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:erbium-alpine
 # this package has dependencies (iconv) that require build tools
 # --no-cache: download package index on-the-fly, no need to cleanup afterwards
 # --virtual: bundle packages, remove whole bundle at once, when done
-RUN apk --no-cache --virtual build-dependencies add python make g++
+RUN apk --no-cache --virtual build-dependencies add python3 make g++
```

Comment:
python no longer exists.  I have to set python2 or python3.  I went with python3